OSDOCS#16913: CQA for getting started with the CLI (#113944)
openshift/openshift-docs: modules/cli-installing-cli-web-console-macos.adoc

$ echo $PATH
----

.Verification

> * After you install the OpenShift CLI, it is available using the `oc` command:

Reviewer: ```suggestion
* After you install the OpenShift CLI, you can use the `oc` command:
```
